Makefile

```diff
@@ -37,7 +37,7 @@ deps.update:
 
 .PHONY: validate
 validate:
-	@if [ -z "${vendor-id}" ]; then $(NPM) run validate; else $(NPM) run validate -- --vendor-id $(vendor-id); fi
+	@if [ -z "${VENDOR_ID}" ]; then $(NPM) run validate; else $(NPM) run validate -- --VENDOR_ID $(VENDOR_ID); fi
```

Comment:
The script argument is still `vendor-id`

Reply:
Huh no i just changed it to `VENDOR_ID`?

Reply:
Let me rephrase: the script argument should still be `vendor-id`, that is idiomic for flags.

Reply:
Ok i *think* i understand it now. Pushed another fix.